core/arch/arm/kernel/otp_stubs.c

```diff
@@ -27,3 +27,20 @@ __weak int tee_otp_get_die_id(uint8_t *buffer, size_t len)
 
 	return 0;
 }
+
+/*
+ * Override this API on your platform to provide TA encryption key as
+ * per your security requirements. There can be two options for this key:
+ *
+ * 1) Unique per device encryption key.
+ * 2) Common encryption key.
+ *
+ * The default implementation chooses option (1).
+ */
+__weak int tee_otp_get_ta_enc_key(uint8_t *buffer, size_t len)
```

Comment:
`TEE_Result` is the preferred error type. Any particular reason why it's not used here?

Reply:
No particular reason. Will use `TEE_Result` instead.